### PR TITLE
Expand WoT CG collaboration

### LINKED ISSUE #36: Expand CG collaboration
Current charter has the following item:

> For collaboration on community outreach of the WG reports to increase adoption and implementation, as well as collecting feedback from the global community. Meetings held in English.

We should change such that a stronger collaboration is apparent. My proposal

> For collaboration on community outreach of the WG reports to increase adoption and implementation, as well as collecting feedback from the global community via reports or joint events. Meetings held in English.

### PR DESCRIPTION
**Pull request by egekorkan**
fixes #36 

Should be taken into consideration by WoT JP CG too.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-charter-drafts/pull/41.html" title="Last updated on Feb 28, 2023, 1:57 PM UTC (5805158)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-charter-drafts/41/ab1594d...5805158.html" title="Last updated on Feb 28, 2023, 1:57 PM UTC (5805158)">Diff</a>

### CODE REVIEW

File: wot-wg-2023-draft.html [hunk 1]
```diff
@@ -437,34 +437,34 @@ <h2>Success Criteria</h2>
...
             <dt><a href="https://w3c.github.io/charter-drafts/charter-template.html"><i class="todo">[other name]</i> Working Group</a></dt>
               <dd><i class="todo">[specific nature of liaison]</i></dd>
 -->
-            <dt><a href="https://www.w3.org/community/wot/">Web of Things Community Group</a></dt>
-              <dd>For collaboration on community outreach of the WG reports to increase adoption and implementation, 
-		  as well as collecting feedback from the global community. 
-		  Meetings held in English.
-		  </dd>	
+    <dt><a href="https://www.w3.org/community/wot/">Web of Things Community Group</a></dt>
+      <dd>For collaboration on community outreach of the WG reports to increase adoption and implementation,
```
Comment (relu91) [inline wot-wg-2023-draft.html:464]:
```suggestion
      <dd>For collaboration on community outreach of the WG reports to increase adoption, implementation, and understanding
```

**Review by relu91 — APPROVED** [review]
Changes look good, we can also mention the "educational" aspect of the CG as mentioned in the charter when talking about guidelines and tutorials.

**Comment by mmccool** [comment]
Comments: 
- Something similar needed for WoT-JP, but add "translations"
- "via" should be non-exclusive, add "including"
- is commitment from WG to collaborate with CG - should be written from that point of view
- delete "of the WG reports" - strange English, unnecessary

File: wot-wg-2023-draft.html [hunk 2]
```diff
@@ -437,34 +437,33 @@ <h2>Success Criteria</h2>
...
-              <dd>For collaboration on community outreach of the WG reports to increase adoption and implementation, 
-		  as well as collecting feedback from the global community. 
-		  Meetings held in English.
-		  </dd>	
+    <dt><a href="https://www.w3.org/community/wot/">Web of Things Community Group</a></dt>
+      <dd>For collaboration on community outreach to increase adoption, implementation, and understanding, as well as 
+        collecting feedback from the global community. Meetings held in English.
+    </dd>	
             <dt><a href="https://www.w3.org/community/wot-jp/">Web of Things Japanese Community Group</a></dt>
               <dd>For collaboration on community outreach of the WG reports to increase adoption and implementation,
```
Comment (mmccool) [inline wot-wg-2023-draft.html:468]: To make things consistent...
```suggestion
              <dd>For collaboration on community outreach to increase adoption and implementation, 
```

File: wot-wg-2023-draft.html [hunk 3]
```diff
@@ -437,34 +437,33 @@ <h2>Success Criteria</h2>
...
             <dt><a href="https://w3c.github.io/charter-drafts/charter-template.html"><i class="todo">[other name]</i> Working Group</a></dt>
               <dd><i class="todo">[specific nature of liaison]</i></dd>
 -->
-            <dt><a href="https://www.w3.org/community/wot/">Web of Things Community Group</a></dt>
-              <dd>For collaboration on community outreach of the WG reports to increase adoption and implementation, 
-		  as well as collecting feedback from the global community. 
-		  Meetings held in English.
-		  </dd>	
+    <dt><a href="https://www.w3.org/community/wot/">Web of Things Community Group</a></dt>
+      <dd>For collaboration on community outreach to increase adoption, implementation, and understanding, as well as
```
Comment (mmccool) [inline wot-wg-2023-draft.html:464]: I see "of the WG reports" was removed, as I suggested, and the extra "via" phrase at the end was removed.   I'm good with this version.

**Review by mmccool — APPROVED** [review]
Good to see a start on the policies document, but I'm going to approve this for the changes to the charter.

**Comment by egekorkan** [comment]
@k-toumura and/or @chachamimm could you extend this PR for the part on CG collaboration or create a new one?

**Comment by k-toumura** [comment]
I think Japanese CG and WoT CG can have the same policy, just different target regions and discussion languages.  What do you think, @chachamimm ?

File: policy.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Collaboration Policy
+
+The WG manages collaboration
+
+## WoT Community Group
+
+ - CG report can be taken as feedback input
```
Comment (mlagally) [inline policy.md:7]: Suggest to reword to:
- CG reports can serve as feedback and input to the WG and IG

**Review by mlagally — CHANGES_REQUESTED** [review]
I'm not sure we need a dedicated policy document that only describes the CG.

If we proceed with this approach, we should describe the entire policy / process,
i.e. the flow from IG use cases, requirements, new building blocks etc. to the WG for spec development and the CG for marketing and inbound communication of non-IG/WG members.
Please see detailed suggestions

File: policy.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+-  WoT WG participants can participate in WoT CG Meetups where implementations of WoT are shown
+-  Social media presence should be managed together and coordinated
```
[the remark this answers is not in the document]
Comment (egekorkan) [reply policy.md:9]: Good point and we need to clarify this. Also see https://github.com/w3c/wot-marketing/issues/362#issuecomment-1438435424

File: policy.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+## WoT Japanese Community Group
+
+To be specified
```
[the remark this answers is not in the document]
Comment (egekorkan) [reply policy.md:13]: Japanese CG also does translations. There might be some organizational differences (website, meeting tool, github usage etc) but our goals are aligned.

File: policy.md (hunk 4 above)
Comment (egekorkan) [reply policy.md:7]:
```suggestion
 - CG reports can serve as feedback and input to the WG and IG
```

**Review by mmccool — CHANGES_REQUESTED** [review]
Please add content for the JP CG or mark with with a "todo" class so it is highlighted.

File: wot-wg-2023-draft.html [hunk 7]
```diff
@@ -470,17 +470,15 @@ <h3 id="w3c-coordination">W3C Groups</h3>
...
-		  as well as collecting feedback from the Japanese community. 
-		  Meetings held in Japanese.
-		  </dd>	
-            <dt><a href="https://www.w3.org/groups/ig/wot">Web of Things Interest Group</a></dt>
+      <dt><a href="https://www.w3.org/community/wot/">Web of Things Community Group</a></dt>
+        <dd>For collaboration on community outreach to increase adoption, implementation, and understanding, as well as collecting feedback from the global community. 
+          Meetings held in English.
+      </dd>	
+      <dt><a href="https://www.w3.org/community/wot-jp/">Web of Things Japanese Community Group</a></dt>
+        <dd>For collaboration on community outreach to increase adoption and implementation, as well as collecting feedback from the Japanese community.
```
Comment (mmccool) [inline wot-wg-2023-draft.html:478]:
```suggestion
        <dd>For collaboration on community outreach to increase adoption, understanding, and implementation, as well as collecting feedback from the Japanese community. 
```

File: policy.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
+# Collaboration Policy
```
Comment (mmccool) [inline policy.md:1]:
```suggestion
# Collaboration Policy
Following is a DRAFT and additional discussion is needed.  This is not binding.
```

**Pull request merged**